clients/go/apis/v1alpha1/config.go

```diff
@@ -64,6 +64,12 @@ type Config struct {
 	// If fail-close is not chosen, the audit logger will log errors that occur,
 	// and then continue without impeding the application in any way.
 	LogMode string `yaml:"log_mode,omitempty" env:"LOG_MODE,overwrite"`
+
+	// JVSEndpoint is the endpoint where public keys may be retrieved from the JVS.
+	// These will be used to validate JWT tokens that are passed in through the
+	// "justification_token" header. If omitted, justifications will not be added
+	// to logs.
+	JVSEndpoint string `yaml:"jvs_endpoint,omitempty" env:"JVS_ENDPOINT,overwrite"`
```

Comment:
Agreed, but it's still pretty common to have a default here. Like, Vault defaults to `http://127.0.0.1:8080` even though Vault is an external server.

Reply:
One additional reason (and i think the stronger one) not to use a default here is that this is really being used for 2 purposes - to set the endpoint, yes, but also as a flag on whether to enable justifications. If we don't set a value for this, the JVS Client isn't initialized, and then we don't try to add justifications to the payload (related to the place where you asked why it could be nil). This is because i made the assumption that we may still want to be able to run Lumberjack without the JVS set up in some cases.

Reply:
That feels like a _really_ dangerous API. I think it's very unexpected that specifying an address would enable this feature.

Reply:
do you think its worth having a separate flag then (EnableJustification?) for the switch on whether to enable or disable justification addition?